python/tvm/script/scope_handler.py

```diff
@@ -282,6 +282,12 @@ def block(axes=None, name_hint: str = "", span: Optional[Span] = None):
                 if block_info.writes
                 else []
             )
+
+            region_detect_mask: int = (block_info.reads is None) | ((block_info.reads is None) << 1)
+            annotations = {} if block_info.annotations is None else block_info.annotations
+            if region_detect_mask != 0:
+                annotations["script_detect_access"] = region_detect_mask
```

Comment:
script_parsing_detect_access

Reply:
Add it to https://github.com/apache/tvm/blob/main/include/tvm/tir/stmt.h#L1321 as a constant. follow new convention(always prefix with tir)